Mjml.Net/Components/Body/MsoButtonComponent.cs

```diff
@@ -0,0 +1,153 @@
+﻿using Mjml.Net.Extensions;
+using Mjml.Net.Types;
+
+namespace Mjml.Net.Components.Body
+{
+    public partial class MsoButtonComponent : ButtonComponent
+    {
+        private Dictionary<string, string> BorderstyleAdapter = new()
+        {
+            { "solid", "Solid" },
+            { "dotted", "Dot" },
+            { "dashed", "Dash" },
+            { "double", "Solid" },
+            { "groove", "Solid" },
+            { "ridge", "Solid" },
+            { "inset", "Solid" },
+            { "outset", "Solid" },
+            { "none", "Solid" },
+            { "hidden", "Solid" }
+        };
+        public override string ComponentName => "mj-msobutton";
+
+        [Bind("mso-proof", BindType.Boolean)]
+        public string? MsoProof = "false";
+
+        [Bind("mso-height", BindType.PixelsOrPercent)]
+        public string? MsoHeight;
+
+        [Bind("mso-width", BindType.PixelsOrPercent)]
+        public string? MsoWidth;
+
+        public override void Render(IHtmlRenderer renderer, GlobalContext context)
+        {
+            var isMsoProof = MsoProof == "true";
+            if (isMsoProof)
+            {
+                RenderMso(renderer);
+                renderer.Content("<!--[if !mso]><!-->");
+                {
+                    base.Render(renderer, context);
+                }
+                renderer.Content("<!--<![endif]-->");
+            }
+            else
+            {
+                base.Render(renderer, context);
+            }
+        }
+
+        private void RenderMso(IHtmlRenderer renderer)
+        {
+            var borderAttributes = new[] { "0pt", "Solid", "#000000" };
+            var stroked = Border != "none";
+            var hasBackgroundColor = !string.IsNullOrEmpty(BackgroundColor) && BackgroundColor != "none";
```

Comment:
I always use IsNullOrWhiteSpace whenever possible.

Reply:
updated to use `IsNullOrWhiteSpace`